src/main/java/no/nav/veilarbperson/client/pdl/domain/PersonNavn.java

```diff
@@ -0,0 +1,13 @@
+package no.nav.veilarbperson.client.pdl.domain;
```

Comment:
Denne ligger i feil mappe. Siden denne klassen ikke direkte blir brukt av PDL, men blir brukt i PersonV2Controller så burde den ligge under no.nav.veilarbperson.domain. Egentlig så kunne man brukt PersonNavn som allerede ligger der hvis det er ingen som har bruk for feltet `forkortetNavn`, så kan man heller vurdere å lage en PersonNavnV2 hvis man trenger å ha ekstra informasjon fra PDL